amworkflow/gcode/log_ploter.py

```diff
@@ -0,0 +1,110 @@
+import gradio as gr
```

Comment:
For what is that?

Reply:
It is a well developed module which can create a webUI rapidly, originally for AI oriented applications but also works fine with our purposes. I know you disagree on adding new GUI but could you simply have a look on the interface? I thought it would be a great feature having a lot of panels in one page for previewing instead of popping out matplotlib windows. 

To use the module simply: 
```bash
pip install gradio
```
And then run the script. It will give you a local ip address leading to the page. For now there is only two panels but I want to show you the possibilities. Of course If you still don't like it I would delete it

Reply:
I will look into it. But I stil think we should focus on the other things first!!

Reply:
Or can we move the things only necessary for visualzation in an own branch

Reply:
I tried the gradio module. You are right, it's  a nice tool. But still I would like to move that file into a new issue/branch (e.g. visualization) and not focusing it in the moment.
first because there are othe rmore important stuff to do
second we will not be able to launch the tool other then local, which limits it use